sklearn/ensemble/voting_classifier.py

```diff
@@ -243,7 +243,12 @@ def predict(self, X):
 
     def _collect_probas(self, X):
         """Collect results from clf.predict calls. """
-        return np.asarray([clf.predict_proba(X) for clf in self.estimators_])
+        t = []
+        for clf in self.estimators_:
+            t3 = clf.predict_proba(X).shape
```

Comment:
I don't see how this helps, and it performs predict_proba twice, which is expensive.

Reply:
Thnx. I will resolve this predict_proba being called twice.
For how this helps, the results returned by KerasClassifier needs to be converted into a column vector, like the other ScikitClassifiers. I though that it can contribute to further generalize the way VotingClassifier can accept the information irrespective of the data structure in which that information is packed (as was shown in the error line ((72,1)..(72, ) ).